src/App.vue

```diff
@@ -23,6 +23,18 @@
     @select="onSelect"
     @update:text="onChangeText"
   />
+
+  <h2>With group ordering and additional groups</h2>
+  <picker
+    :additional-groups="additionalGroups"
+    :group-names="{
+      my_custom_group: 'Frequently used',
+      smileys_people: 'Smileys!',
+    }"
+    :group-order="['my_custom_group', 'smileys_people', 'flags']"
+    :group-icons="{ my_custom_group: custom }"
```

Comment:
To make it more customizable and consistent, I think we can set these props/values to the store as other options. Example:


https://github.com/delowardev/vue3-emoji-picker/blob/6a42d8d9ffd63a0208a091916492fb56a9d85801/src/components/Picker.vue#L105-L117

Access options from the store:
https://github.com/delowardev/vue3-emoji-picker/blob/6a42d8d9ffd63a0208a091916492fb56a9d85801/src/components/Body.vue#L82

Reply:
I had initially designed it this way however it seems like the store is shared between pickers. I tried it again here to see if I did anything wrong but I feel like the store is shared somehow. I can see that the `native` picker is correct due to it not being reactive on load. 

https://github.com/delowardev/vue3-emoji-picker/blob/6a42d8d9ffd63a0208a091916492fb56a9d85801/src/components/Body.vue#L126

Any update to the `state.options.native` value isn't updated because it's not wrapped in a `computed` property.

See my screenshot below printing the `store.options` above the picker.

![image](https://user-images.githubusercontent.com/14275291/196781411-49e48d74-1b42-428c-af7b-c2e161be355d.png)

Reply:
I pushed my changes constated to a different branch. Check https://github.com/delowardev/vue3-emoji-picker/compare/main...maarteNNNN:sort-groups-store-mockup.

Reply:
It must be a bug, I will check this one tomorrow. Thanks

Reply:
Thanks @delowardev if there's anything out of order from my side let me know!

Reply:
Hi @delowardev . Any luck with this? I'm hoping we can get this merged in and start using it in our app soon...

Reply:
Hi @scottward, I will merge and push these features tomorrow.

Reply:
@maarteNNNN there was a mistake in `store/index.ts#L172` that was causing the reactivity issue,

https://github.com/delowardev/vue3-emoji-picker/blob/05f1a9795b3b36d936cb91767b7cb4c10f0ae335/src/store/index.ts#L172

which should be `state.options = Object.assign({}, state.options, options)`